db/migrate/20200421195048_create_movies.rb

```diff
@@ -1,15 +0,0 @@
-class CreateMovies < ActiveRecord::Migration[6.0]
```

Comment:
Why was this migration deleted? You should leave this migration (that already run) and add a new one to create `lists` table.